hosts/hetzner-ephemeral-beta/default.nix

```diff
@@ -0,0 +1,77 @@
+{ pkgs, config, inputs, lib, ... }:
+{
+  # User options
+  users = {
+    juuso.authorizedKeys = [ "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBNMKgTTpGSvPG4p8pRUWg1kqnP9zPKybTHQ0+Q/noY5+M6uOxkLy7FqUIEFUT9ZS/fflLlC/AlJsFBU212UzobA= ssh@secretive.sandbox.local" ];
+    kari.authorizedKeys = [ "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIKEdpdbTOz0h9tVvkn13k1e8X7MnctH3zHRFmYWTbz9T kari@torque" ];
+  };
+
+  # Users group
+  users.groups.users = { };
+
+  # Localization
+  networking.hostName = "hetzner-ephemeral-beta";
+  time.timeZone = "Europe/Helsinki";
+
+  # Support for cross compilation
+  boot.binfmt.emulatedSystems = [
+    "aarch64-linux"
```

Comment:
This is unnecessary and possibly problematic as well.

Reply:
Removed this, this probably was the cause of the boot process hanging.